lib/rules/jsx-handler-names.js

```diff
@@ -63,7 +65,7 @@ module.exports = {
         } else if (propFnIsNamedCorrectly && !propIsEventHandler) {
           context.report({
             node: node,
-            message: `Prop key for ${propValue} must begin with '${eventHandlerPropPrefix}'`
+            message: `Prop key for ${propValue} must begin with '${eventHandlerPropRegex}'`
```

Comment:
This isn't very clear; "begin with" only makes sense when followed by a string. If it's a regex (which it shouldn't be) then it'd need something like "match pattern".

What about "Prop key for ${propValue} (matches pattern ${eventHandlerPropRegex}) must begin with ${eventHandlerPropPrefix}"?